easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-5.2.2.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'PythonBundle'
+
+name = 'vsc-mympirun'
+version = '5.2.2'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """mympirun is a tool to make it easier for users of HPC clusters to
+run MPI programs with good performance."""
+
+# we build this to work with every python version
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+use_pip = False
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('setuptools', '41.6.0', {
```

Comment:
Please add the previous comment about the restriction on `setuptools`, if `vsc-install` v0.17 still needs `setuptools` < 42.0.